db/schema.rb

```diff
@@ -230,12 +230,12 @@
     t.decimal "public_dashboard_alert_weighting", default: "5.0"
     t.decimal "teacher_dashboard_alert_weighting", default: "5.0"
     t.decimal "find_out_more_weighting", default: "5.0"
+    t.text "find_out_more_table_variable", default: "none"
```

Comment:
Not part of this merge

Reply:
I'm confused by this! If I run db:migrate on production database on master, this variable moves in the db/schema.rb !